fpsim/locations/kenya/data/method_time_coefficients.csv

```diff
@@ -1,64 +1,46 @@
-estimate,coef,method,functionform
-shape,0.279355915,Injectable,gamma
-rate,-2.999804903,Injectable,gamma
-"age_grp_fact(0,18]",0.50088332,Injectable,gamma
-"age_grp_fact(18,20]",0.207710694,Injectable,gamma
-"age_grp_fact(20,25]",0,Injectable,gamma
-"age_grp_fact(25,35]",-0.455048528,Injectable,gamma
-"age_grp_fact(35,50]",-0.716333164,Injectable,gamma
-meanlog,2.11517825,None,lognormal
-sdlog,0.254967479,None,lognormal
-"age_grp_fact(0,18]",-0.162425317,None,lognormal
-"age_grp_fact(18,20]",0.091653353,None,lognormal
-"age_grp_fact(20,25]",0,None,lognormal
-"age_grp_fact(25,35]",0.128964912,None,lognormal
-"age_grp_fact(35,50]",1.021583111,None,lognormal
-meanlog,2.966699812,Condom,lognormal
-sdlog,0.612927808,Condom,lognormal
-"age_grp_fact(0,18]",-0.94435802,Condom,lognormal
-"age_grp_fact(18,20]",-0.295970987,Condom,lognormal
-"age_grp_fact(20,25]",0,Condom,lognormal
-"age_grp_fact(25,35]",0.637698125,Condom,lognormal
-"age_grp_fact(35,50]",1.041439262,Condom,lognormal
-shape,0.333113131,Implant,gamma
-rate,-3.826711563,Implant,gamma
-"age_grp_fact(0,18]",0.561569578,Implant,gamma
-"age_grp_fact(18,20]",0.566440527,Implant,gamma
-"age_grp_fact(20,25]",0,Implant,gamma
-"age_grp_fact(25,35]",-0.342687222,Implant,gamma
-"age_grp_fact(35,50]",-1.085082328,Implant,gamma
-shape,0.573198596,IUD,gamma
-rate,-3.179136321,IUD,gamma
-"age_grp_fact(0,18]",2.04758411,IUD,gamma
-"age_grp_fact(18,20]",0.318580688,IUD,gamma
-"age_grp_fact(20,25]",0,IUD,gamma
-"age_grp_fact(25,35]",-0.484351121,IUD,gamma
-"age_grp_fact(35,50]",-1.364533532,IUD,gamma
-meanlog,2.359461879,Pill,lognormal
-sdlog,0.264891462,Pill,lognormal
-"age_grp_fact(0,18]",-0.6271321,Pill,lognormal
-"age_grp_fact(18,20]",-0.57164622,Pill,lognormal
-"age_grp_fact(20,25]",0,Pill,lognormal
-"age_grp_fact(25,35]",0.509506377,Pill,lognormal
-"age_grp_fact(35,50]",1.098985677,Pill,lognormal
-shape,0.76407859,Withdrawal,llogis
-scale,2.501787334,Withdrawal,llogis
-"age_grp_fact(0,18]",-0.5755443,Withdrawal,llogis
-"age_grp_fact(18,20]",0.300416984,Withdrawal,llogis
-"age_grp_fact(20,25]",0,Withdrawal,llogis
-"age_grp_fact(25,35]",0.553921344,Withdrawal,llogis
-"age_grp_fact(35,50]",1.355307265,Withdrawal,llogis
-shape,0.345643206,Other.trad,gamma
-rate,-2.928180051,Other.trad,gamma
-"age_grp_fact(0,18]",0.450960888,Other.trad,gamma
-"age_grp_fact(18,20]",0.197750574,Other.trad,gamma
-"age_grp_fact(20,25]",0,Other.trad,gamma
-"age_grp_fact(25,35]",-0.318908929,Other.trad,gamma
-"age_grp_fact(35,50]",-0.972883241,Other.trad,gamma
-meanlog,1.786422667,Other.mod,lognormal
-sdlog,0.422155552,Other.mod,lognormal
-"age_grp_fact(0,18]",-0.397191956,Other.mod,lognormal
-"age_grp_fact(18,20]",-1.786422667,Other.mod,lognormal
-"age_grp_fact(20,25]",0,Other.mod,lognormal
-"age_grp_fact(25,35]",1.162085828,Other.mod,lognormal
-"age_grp_fact(35,50]",2.282028383,Other.mod,lognormal
\ No newline at end of file
+"estimate","coef","se","method","functionform","country"

```

Comment:
Looks like the age groups have changed, is that right? Is it possible to revert to the old age groups? It would make things easier if we had the same age groups for the switching matrices and the durations.

Reply:
Oh yes will do that now.